src/main/java/lotto/Monitor.java

```diff
@@ -0,0 +1,48 @@
+package lotto;
+import camp.nextstep.edu.missionutils.Console;
+import java.util.Arrays;
+import java.util.List;
+
+
+public class Monitor {
+
+
+    public void inputMoney(LottoVariable lottoVariable) {
+        System.out.println("구입금액을 입력해 주세요.");
+        lottoVariable.inputmoney = Integer.parseInt(Console.readLine()); // 금액 입력
+    }
+    public void WinningNumbers(LottoVariable lottoVariable) {
+        System.out.println("당첨 번호를 입력해 주세요.");
+        lottoVariable.selectNumber.addAll(
+                Arrays.stream(Console.readLine().split(",")).map(Integer::parseInt)
+                        .toList());
+    }
```

Comment:
```suggestion
    public void WinningNumbers(LottoVariable lottoVariable) {
        System.out.println("당첨 번호를 입력해 주세요.");
        lottoVariable.selectNumber.addAll(
                Arrays.stream(Console.readLine().split(","))
                .map(Integer::parseInt)
                        .toList()
                        );
    }
```
라인을 구분하면 가독성이 더 좋을 것 같다~

Reply:
넵 확인했습니다!